packages/paste-website/src/components/icons-list/IconCard.tsx

```diff
@@ -44,8 +44,8 @@ const IconCard: React.FC<IconCardProps> = ({selectedIcon}) => {
           </Box>
         </Box>
         <Paragraph marginBottom="space0">
-          Be sure to read the <SiteLink to="/getting-started/quickstart">quick start guide</SiteLink> page to ensure you
-          have the correct dependencies.
+          Be sure to read the <SiteLink to="/introduction/for-engineers/quickstart">quick start guide</SiteLink> page to
```

Comment:
It could be helpful, but I think it should be a separate ticket. We'd need to flesh out how more details, like all links or only links that are used multiple times?